apps/qubit/modules/term/templates/indexSuccess.php

```diff
@@ -254,11 +254,12 @@
           <?php endif; ?>
 
         <div id="sort-header">
-          <?php echo get_partial('default/sortPicker',
-            array(
-              'options' => array(
-                'lastUpdated' => __('Most recent'),
-                'alphabetic' => __('Alphabetic')))) ?>
+          <?php echo get_partial('default/sortPicker', array(
+            'options' => array(
+              'lastUpdated' => __('Most recent'),
+              'alphabetic'  => __('Alphabetic'),
+              'identifier'  => __('Reference code'),
+              'date'        => __('Date')))) ?>
```

Comment:
the ref code and date case is handled here in the linked descriptions, just like the regular descriptions browse page: https://github.com/artefactual/atom/blob/qa/2.4.x/apps/qubit/modules/informationobject/actions/browseAction.class.php#L527